### DISCUSSION TITLE
DetailsList & Managed Metadata

### DISCUSSION THREAD

**Discussion by Hillgard** [microsoft/powercat-creator-kit#47] [Q&A] [answered]
Hello,

in the Detailslist, control, how can we set a managed metadata value in a column ?

Example:
Table(
    {  
ColName:"infotopic",ColDisplayName:"Topic",ColCellType:"",ColSortable:true,ColWidth:1000,ColRowHeader:true,ColResizable:true
    }
)

We see the @data_type as a text in the column:

![image](https://user-images.githubusercontent.com/103180655/174249755-b551b3ed-1891-4fcd-bacb-68c5289d449c.png)

My question: how can i get the "Label" property ?

Thank you in advance
Hill

**Comment by Ramakrishnan24689** [comment, marked as answer]
Hi @Hillgard,

Thanks for reaching out.
Since Managed metadata value comprises of an Object, one way of getting the **Label** property is to customize the Input/Items collection by adding a new column which contains Label value. So, the **Items** property of the detail list will look like this

`AddColumns('SharePointList',"InfotopicsLabel",infotopic.Label)`

and then **columns_Items** property can refer this new column once it is included as part of the data.

`Table(
    {
        ColName: "InfotopicsLabel",
        ColDisplayName: "Topic",
        ColSortable: true,
        ColWidth: 200,
        ColRowHeader: true,
        ColResizable: true
    }
)`

**Reply by Hillgard** [reply]
Hello @Ramakrishnan24689 ,

Thank you for your answer, it's working fine with this way.
Just added the new column in the "Fields" property of the detailsList and it's working fine.

Thank you for your assistance
Hill
